.ci/docker/common/install_triton.sh

```diff
@@ -19,7 +19,7 @@ elif [ -n "${XPU_VERSION}" ]; then
   TRITON_REPO="https://github.com/intel/intel-xpu-backend-for-triton"
   TRITON_TEXT_FILE="triton-xpu"
 else
-  TRITON_REPO="https://github.com/openai/triton"
+  TRITON_REPO="https://github.com/nmacchioni/triton"
```

Comment:
What Triton changes are needed to support this diff?  Can you link a PR or something in the summary?

Reply:
Here is probably the best place to brainstorm how we could upstream these changes to Triton, I figured our best chance would be adding a switch to Triton's do_bench that falls back to our benchmarker instead.

Reply:
Once we're approved here we can work with OpenAI to make this accessible from Triton, stack whatever that change ends up being on top of the release branch and bump the Triton commit hash by one to that PR. This should be safe in OSS (assuming all tests are passing, and after I do a final performance run), and internally (everything is gated by JKs and the current default is to fallback to the original benchmarking method), but of course we will test there as well.